### PR TITLE
remove use_smap + fix pipe test

### PR DESCRIPTION
**Pull request by Lenoush**
- use compute_smaps instead of use_smap in the init of MRITensorflowNUFFT
- fix nm_iter into num_iterations in the pipe function 
- change syntaxe of pipe test because tensorflow mri  don't use osf (oversampling factor)

### CODE REVIEW

File: tests/operators/test_density_for_op.py
Comment (paquiteau) [inline tests/operators/test_density_for_op.py:36]:
```suggestion
    if backend == "tensorflow" and osf != 1:
        pytest.skip("Tensorflow does not support OSF != 1")
    result = pipe(traj, shape, backend, osf=osf, num_iterations=10)
```

Comment (paquiteau) [reply tests/operators/test_density_for_op.py:36]: just simplifying a bit.

**Review by paquiteau — CHANGES_REQUESTED** [review]
Nice, do you wish to have this run in the CI as well ?

File: tests/operators/test_density_for_op.py
Comment (Lenoush) [reply tests/operators/test_density_for_op.py:36]: i thought of doing this but tensorflow mri doesn't take osf arguments: so to avoid having an unused argument in pipe i added conditions in test

Comment (Lenoush) [reply tests/operators/test_density_for_op.py:36]: we can add this in the CI , i test on my machine and it work

Comment (paquiteau) [reply tests/operators/test_density_for_op.py:36]: > 

i thought of doing this but tensorflow mri doesn't take osf arguments: so to avoid having an unused argument in pipe i added conditions in test

In this case, I would recommend having osf=1 as a dummy argument in tfnufft pipe (and raise an error there). This would make a consistent API call for all pipe methods.

Comment (chaithyagr) [reply tests/operators/test_density_for_op.py:36]: By default osf=2 internally within `tensorflow-mri` I recommend using the same default value.

**Comment by chaithyagr** [comment]
I would wait for #160 to merge to master before merging this PR. We can see if tests are green then.

**Comment by chaithyagr** [comment]
Closing as this is now taken in #160, which also does the tests for tensorflow-nufft.

**Pull request closed without merging**
